Formula/sdl.rb

```diff
@@ -67,6 +67,8 @@ class Sdl < Formula
   # SDL 1.2 is deprecated, unsupported, and not recommended for new projects.
   deprecate! date: "2013-08-17", because: :deprecated_upstream
 
+  conflicts_with "sdl12-compat", because: "sdl12-compat is a drop-in replacement for sdl"
```

Comment:
We could probably add the `conflicts_with` to both formulae in a separate syntax-only PR to avoid having to rebuild `sdl`.

Reply:
I'm not sure how to do that; I'm not great with advanced git.

Reply:
Just remove the `conflicts_with` from both formulae and then squash all your commits here together.